pydra/engine/tests/test_task.py

```diff
@@ -1028,21 +1047,92 @@ def test_audit_shellcommandtask(tmpdir):
     shelly()
     message_path = tmpdir / shelly.checksum / "messages"
     # go through each jsonld file in message_path and check if the label field exists
-    label_content = []
+
     command_content = []
 
     for file in glob(str(message_path) + "/*.jsonld"):
         with open(file, "r") as f:
             data = json.load(f)
-            print(data)
-            if "Label" in data:
-                label_content.append(True)
+
+            if "@type" in data:
```

Comment:
I believe all messages should have `@type`, am I right?

Reply:
WIP